package.json

```diff
@@ -107,7 +107,7 @@
       "./scripts/jest/environment.js"
     ],
     "setupTestFrameworkScriptFile": "./scripts/jest/test-framework-setup.js",
-    "testRegex": "/__tests__/",
+    "testRegex": "/__tests__/.*\\.js$",
```

Comment:
This fixes an issue that caused Jest to treat snapshot files as test suites. Also matches Jest docs.